xml/monitoring.xml

```diff
@@ -148,24 +148,23 @@
    <title>Using the Ganglia Web interface</title>
    <para>
     Install <package>ganglia-web</package> on the management server.
-    Depending on which PHP version is used (default is PHP 5), enable it in
-    Apache2: <command>a2enmod php5</command> or <command>a2enmod
+    Depending on which PHP version is used (the default is PHP 7), enable it in
+    Apache2: <command>a2enmod php7</command> or <command>a2enmod
     php7</command>. Then start Apache2 on this machine: <command>rcapache2
     start</command> and make sure it is started automatically after a
     reboot: <command>systemctl enable apache2</command>. The Ganglia Web
     interface should be accessible from
-    <literal>http://<replaceable>MANAGEMENT_SERVER</replaceable>/ganglia</literal>.
+    <literal>http://<replaceable>MANAGEMENT_SERVER</replaceable>/ganglia-web</literal>.
```

Comment:
>  maybe reformat directly on main

Oh! I did not expect that. 

I did not previously notice that this one file has inconsistent formatting, both within itself and relative to the others. 

OK, point noted. It will make it harder, but that is doable.

>  did you copy the new path from the release notes

Yes, that is what I did, and changed PHP5 to PHP7. I do not have access to an HPC cluster to try it.

Reply:
> OK, point noted. It will make it harder, but that is doable.

I prefixed that suggestion with "Ideally, " for a reason. No need to beat yourself up about it. :)

> Yes, that is what I did, and changed PHP5 to PHP7. I do not have access to an HPC cluster to try it.

I installed the ganglia-web package on Leap 15.2, and it appears to run on `localhost/ganglia` by default, afaict (@e4t ?):

```
o /etc/apache2/conf.d/ganglia-web.conf

[...]
Alias /ganglia /srv/www/htdocs/ganglia-web

<Location /ganglia>
  Require all granted
  # Require host example.org
</Location>
```